jvm/src/main/scala/io/glutenproject/execution/BasicPhysicalOperatorTransformer.scala

```diff
@@ -135,35 +166,28 @@ case class FilterExecTransformer(
 
   // override def canEqual(that: Any): Boolean = false
 
-  def getRelNode(args: java.lang.Object, childRel: RelNode,
-                 validation: Boolean = false): RelNode = {
-    prepareFilterRel(args, condition, child.output, childRel, validation)
-  }
-
   override def doTransform(context: SubstraitContext): TransformContext = {
+    val leftCondition = getLeftCondition
     val childCtx = child match {
       case c: TransformSupport =>
         c.doTransform(context)
       case _ =>
         null
     }
-    if (GlutenConfig.getConf.isVeloxBackend && child.isInstanceOf[BatchScanExecTransformer]) {
+    if (leftCondition == null) {
+      // The computing for this filter is not needed.
       return childCtx
```

Comment:
Pushed down the filters in to Scan, so for Velox this FilterExec can be ignored. Does CH depends on the FilterExec here?

Reply:
> @zzcclp Pushed down the filters in to Scan, so for Velox this FilterExec can be ignored. Does CH depends on the FilterExec here?

CH depends on the FilterExec to do filters, it can not support using `prewhere` when scanning.